cli/src/commands/get/streams.rs

```diff
@@ -131,46 +139,139 @@ impl DisplayTable for StreamStat {
             } else {
                 "none".dimmed()
             },
+            if let Some(precision) = self.maintain_recall_precision {
+                red_if_lower_green_otherwise(precision, self.precision)
+            } else {
+                "none".dimmed()
+            },
+            if let Some(recall) = self.maintain_precision_recall {
+                red_if_lower_green_otherwise(recall, self.recall)
+            } else {
+                "none".dimmed()
+            },
+            if let Some(threshold) = self.maintain_recall_threshold {
+                format!("{:.5}", threshold).normal()
+            } else {
+                "none".dimmed()
+            },
+            if let Some(threshold) = self.maintain_precision_threshold {
+                format!("{:.5}", threshold).normal()
+            } else {
+                "none".dimmed()
+            }
         ]
     }
 }
 
 fn red_if_lower_green_otherwise(test: NotNan<f64>, threshold: NotNan<f64>) -> ColoredString {
     let test_str = format!("{:.3}", test);
 
+    let diff = test - threshold;
+
     match test {
-        test if test < threshold => format!("{test_str} (decrease)").red(),
-        test if test > threshold => format!("{test_str} (increase)").green(),
+        test if test < threshold => format!("{test_str} ({diff:.3})").red(),
+        test if test > threshold => format!("{test_str} (+{diff:.3})").green(),
```

Comment:
You can use the printf directive `+` to always add the appropriate string here:

```rust
fn main() {
    for value in [0.0, -0.0, 0.0, -42.3, 42.3] {
        println!("{:+0.3}", value);
    }
}
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=54b7555880b48aa31f24fd5f3fc499e1